main.js

```diff
@@ -0,0 +1,32 @@
+// eslint-disable-next-line no-unused-vars
+const openmenu = document.querySelector('#button-hamburger');
+openmenu.addEventListener('click', () => {
+  const navbarOptions = document.querySelector('#navbar-options');
+  navbarOptions.classList.replace('nav-menu', 'menu-mobile');
+  const closeIcon = document.querySelector('#x');
+  closeIcon.classList.replace('hidden', 'close-icon-visible');
+});
+
+const closemenu = document.querySelector('#x');
+closemenu.addEventListener('click', () => {
+  const navbarOptions = document.querySelector('#navbar-options');
+  navbarOptions.classList.replace('menu-mobile', 'nav-menu');
+});
+
+const closeLinkP = document.querySelector('#links-p');
+closeLinkP.addEventListener('click', () => {
+  const navbarOptions = document.querySelector('#navbar-options');
+  navbarOptions.classList.replace('menu-mobile', 'nav-menu');
+});
+
+const closeLinkA = document.querySelector('#links-a');
+closeLinkA.addEventListener('click', () => {
+  const navbarOptions = document.querySelector('#navbar-options');
+  navbarOptions.classList.replace('menu-mobile', 'nav-menu');
+});
+
+const closeLinkC = document.querySelector('#links-c');
+closeLinkC.addEventListener('click', () => {
+  const navbarOptions = document.querySelector('#navbar-options');
+  navbarOptions.classList.replace('menu-mobile', 'nav-menu');
```

Comment:
- According to the DRY principles, in order to keep your code clean, it is necessary to avoid repetitions as much as possible.  These lines of code` const navbarOptions = document.querySelector('#navbar-options');
  navbarOptions.classList.replace('menu-mobile', 'nav-menu');`, were repeated quite a few times. I suggest you create a named function for those lines and call the function when required. Please try to fix this 🙏 🙏